charts/values.yaml

```diff
@@ -78,19 +78,33 @@ persistence:
 extraVolumes: []
 # specify extra list of volumeMounts
 extraVolumeMounts: []
-# specify extra list of containerPorts
-extraContainerPorts: []
+
+## Container listener port configuration
+## ref: https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/
+## Warning: Changing port on an existing cluster will lead to pod rolling update.
+## @param additionalContainerPorts. Additional container port 
+## Example:
+## - name: app1
+##   containerPorts: 8010
+##   protocol: TCP
+additionalContainerPorts: []
+
 
 # Configure the Service to access Marklogic Clusters
 service:
   type: ClusterIP
-  ports:
-    - protocol: TCP
-      name: query-console
-      port: 8000
-    - protocol: TCP
-      name: manage
-      port: 8002
+  ## Service listener port configuration
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/connect-applications-service/
+  ## @param service.additionalPorts. Additional service port 
+  ## Example:
+  ## - name: app1
+  ##   port: 8010 
+  ##   targetPort: 8010
+  ##   protocol: TCP
+  additionalPorts: []
```

Comment:
Move this up above "additionalContainerPorts" as it is more important. "additionalContainerPorts" shouldn't need to be set except for in special cases, right?